pkg/pd/pd.go

```diff
@@ -14,9 +14,13 @@
 package pd
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/joomcode/errorx"
+	"github.com/pingcap/tidb-dashboard/pkg/utils/distro"
 )
 
 var (
-	ErrNS = errorx.NewNamespace("error.pd")
```

Comment:
Maybe this one is fine not to replace? It's not text content to display.

Reply:
That's right. I've some commits lost in the wind, cherry-pick is dangerous!